network/80-wifi-adhoc.network

```diff
@@ -1,5 +1,5 @@
 [Match]
-Type=wifi
+Type=wlan
```

Comment:
hmm, we really should document the list of allowed Types= in .network man page. Currently we don't. We inherit it from the kernel's sysfs devtype attribute, so we cannot be comprehensive, but we should say that and list at least a bunch of common types. @yuwata?